src/components/DropMenu/DropMenu.jsx

```diff
@@ -414,6 +414,7 @@ const DropMenu = createClass({
 
 		if (!optionProps.isDisabled && focusedIndex !== optionIndex) {
 			onFocusOption(optionIndex, { props: optionProps, event });
+			optionProps.onClick({{ props: optionProps, event }});
```

Comment:
It seems weird to fire two events for a single action; should it be a precedence thing? e.g.:
`if optionProps.onClick is set` -> call that function
`else -> ` call `onSelect` function